test/unit/test_intrahost.py

```diff
@@ -1,12 +1,34 @@
 # Unit tests for intrahost.py
 
-__author__ = "PLACEHOLDER"
+__author__ = "irwin@broadinstitute.org"
 
 import intrahost
-import unittest, argparse
+import unittest, argparse, os, tempfile
+import util.file, tools.vphaser
+from test import TestCaseWithTmp
 
 class TestCommandHelp(unittest.TestCase):
     def test_help_parser_for_each_command(self):
         for cmd_name, parser_fun in intrahost.__commands__:
             parser = parser_fun(argparse.ArgumentParser())
             helpstring = parser.format_help()
+
+class TestVPhaser(TestCaseWithTmp) :
```

Comment:
Can you move this class to test/unit/test_tools_vphaser2.py (a file that does not yet exist)? That can test just the Phaser2 wrapper itself and test_intrahost can deal more with custom code, filters, and end-to-end stuff.